babel.config.js

```diff
@@ -1,3 +1,12 @@
 module.exports = {
   presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    "react-native-classname-to-style",
+    [
+        "react-native-platform-specific-extensions",
+        {
```

Comment:
Codacy found an issue: [Expected indentation of 6 spaces but found 8.](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=6016116)